services/submission/src/main/resources/log4j2dev.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
make file names consistent (log4j2-dev.xml)

Reply:
Thank you. Good catch